src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -14,6 +24,6 @@ void npe(String a, String b) {
 
  @Override
  public boolean equals(Object obj) {
```

Comment:
Reporter: PMD
Rule: OverrideBothEqualsAndHashcode
Severity: WARN
File: /home/bjerre/workspace/violation-comments-to-github-plugin/work/jobs/test/workspace/violations-test/src/main/java/se/bjurr/violations/lib/example/MyClass.java L26
 

Basic http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/basic.html#OverrideBothEqualsAndHashcode

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>